lib/src/clojure_lsp/dep_graph.clj

```diff
@@ -0,0 +1,344 @@
+(ns clojure-lsp.dep-graph
```

Comment:
Pretty good functions on this file, each one with its own responsibility and very composable 👏🏻